src/js/components/Notification/stories/Global.js

```diff
@@ -43,10 +43,10 @@ export const Global = () => {
           />
         )}
         <Box width="large" margin="auto" pad="medium" gap="medium">
-          <Header>
+          <Box direction="row" justify="between">
```

Comment:
Instead of changing this from `Header` to `Box` I think we should change the `Box` at line 29 to `Main`. This ensures that this container is still correctly marked as a `Header`, it just won't have the banner role.

Reply:
Great idea! Added the header back and added the `Main`.